lib/kino/explorer.ex

```diff
@@ -161,7 +161,7 @@ defmodule Kino.Explorer do
   end
 
   defp build_summary(:categorical, column, series, has_groups, grouped, nulls) do
-    if compute_summaries?(series) do
+    try do
       %{"counts" => top_freq, "values" => top} = most_frequent(series)
```

Comment:
What if we have the try-catch only around most_frequent and we avoid adding `counts` and `values` to the header if it fails?

Reply:
If you have to keep it around the whole summary, please add a `# TODO` above explaining the exact case that fails, so we can double check it in future polars versions.